plugins/modules/elb_application_lb.py

```diff
@@ -420,22 +430,47 @@
     description: The Amazon Resource Name (ARN) of the load balancer.
     returned: when state is present
     type: str
-    sample: arn:aws:elasticloadbalancing:ap-southeast-2:0123456789:loadbalancer/app/my-elb/001122334455
+    sample: "arn:aws:elasticloadbalancing:ap-southeast-2:0123456789:loadbalancer/app/my-elb/001122334455"
 load_balancer_name:
     description: The name of the load balancer.
     returned: when state is present
     type: str
-    sample: my-elb
+    sample: "my-elb"
+resource_actions:
```

Comment:
I guess there is no need to document `resource_actions:`.

Reply:
I was just covering every parameter returned.. does this mean I should also remove `changed` and `failed` ?

Reply:
Yep don't think other modules have this added here and I don't think its relevant to the module itself per say.